### ISSUE TITLE
Search: Sort results by proximity to address or Zipcode

### ISSUE THREAD

**Issue by bdolly** [CityOfPhiladelphia/ppr-finder#1]
Sorts Search Results Sets by proximity to user address in units of miles

**Comment by bdolly** [comment]
Example Carto DB SQL query to return results sets in order of distance from given address
```SQL
SELECT ppr_programs.*, ST_Y(ST_Centroid(ppr_assets.the_geom)) as latitude, ST_X(ST_Centroid(ppr_assets.the_geom)) as longitude, ST_Distance(
                  ST_Centroid(ppr_assets.the_geom)::geography,
                  ST_SetSRID(
                    ST_Point(-75.16097658476633,39.951661655671955),
                    4326
                  )::geography
                ) * 0.000621371 as distance 
FROM ppr_programs 
INNER JOIN ppr_facilities ON (ppr_facilities.id = ppr_programs.facility->>0) 
INNER JOIN ppr_assets ON (ppr_assets.objectid = ppr_facilities.pprassets_object_id) 
WHERE (program_name ILIKE '%soccer%' OR program_description ILIKE '%soccer%') 
ORDER BY distance ASC
```


Note the addition of the two `::geography` and ` * 0.000621371 `

**Comment by bdolly** [comment]
closed with a09d11e03407b7bf422c81a9878ee9cb76e12297
